src/beacon/app/routes/analytics.py

```
@@ -0,0 +1,481 @@
+    days: int = Query(7, ge=1, le=90, description="Number of days to analyze")
+):
+    """
+    Get data transmission analytics summary
+    """
+    end_date = datetime.utcnow()
+    start_date = end_date - timedelta(days=days)
+    
+    # Get total devices
+    total_devices = db.exec(select(func.count(Device.device_key))).first()
+    active_devices = db.exec(
+        select(func.count(Device.device_key)).where(Device.is_active == True)
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `Device.is_active:` for truth checks

Replace with `Device.is_active`

(E712)